### PR TITLE
Fixed Progress and Input class issue

### PR DESCRIPTION
**Pull request by Shamimkhan321**
Fixed Progress and Input class issue

### CODE REVIEW

File: stories/Atom/Navigation/ProgressBarNavigation/ProgressBarNavigation.scss [hunk 1]
```diff
@@ -5,24 +5,32 @@
...
-  height: 12px;
+  height: 0.75rem;
   width: 100%;
 
   @include devicebreak(medium) {
-    height: 16px;
+    height: 1rem;
+  }
+
+  &.small, &.small .progress-bar{
```
Comment (shishirphadke) [inline stories/Atom/Navigation/ProgressBarNavigation/ProgressBarNavigation.scss:15]: Hey @Shamimkhan321 Can you write this as:
```
&.small {
 // Styles...

  .progress-bar { 
    // Styles... 
  }
}
```

File: stories/Atom/Navigation/ProgressBarNavigation/ProgressBarNavigation.scss [hunk 2]
```diff
@@ -5,24 +5,32 @@
...
+    height: 0.625rem;
+    
+    @include devicebreak(medium) {
+      height: 0.75rem;
+    }
   }
 }
 
 .progress-bar {
   background: $color-yellow;
```
Comment (shishirphadke) [inline stories/Atom/Navigation/ProgressBarNavigation/ProgressBarNavigation.scss:25]: @Shamimkhan321 Can we add background color based on modifier?

File: stories/Components/Forms/Inputfields/input_group.scss [hunk 3]
```diff
@@ -4,22 +4,18 @@
...
 
-      &-help {
+    &-help {
+      display: block;
+      font-size: $font-size-14;
+      margin-bottom: 0;
+    }
+  }
+  
+  &.disabled{
```
Comment (shishirphadke) [inline stories/Components/Forms/Inputfields/input_group.scss:16]: Add whitespace before `{`.

File: stories/Components/Forms/Inputfields/input_group.scss [hunk 4]
```diff
@@ -4,22 +4,18 @@
...
+    .input-group{
+      &__label,&__label-help{
```
Comment (shishirphadke) [inline stories/Components/Forms/Inputfields/input_group.scss:18]: Same as above.

File: stories/Atom/Navigation/ProgressBarNavigation/ProgressBarNavigation.scss (hunk 2 above)
Comment (Shamimkhan321) [reply stories/Atom/Navigation/ProgressBarNavigation/ProgressBarNavigation.scss:25]: there is no specify for background color change.

**Pull request merged**